c2corg_ui/static/js/feed.js

```diff
@@ -114,16 +163,165 @@ app.FeedController = function(appAuthentication, appApi, appLang, imageUrl, ngeo
   this.isPersonal = !this.userId;
 
   /**
+   * @type {boolean}
+   * @export
+   */
+  this.showMobileBlock = /** @type {boolean} */ (JSON.parse(window.localStorage.getItem('showMobileBlock') || 'true'));
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.showAssoBlock = /** @type {boolean} */ (JSON.parse(window.localStorage.getItem('showAssoBlock') || 'true'));
+
+  /**
    * @type {ngeo.Location}
    * @public
    */
   this.ngeoLocation = ngeoLocation;
 
   this.getDocumentsFromFeed();
+  this.getLatestTopics_();
```

Comment:
I think you should add a test on ``this.isPersonal`` before running this function, because it is not wanted for personal (profile) feeds.

Reply:
Actually this function does not make sense to me: the feed directive has nothing to do with the forum, it is only responsible for displaying the guidebook activity cards. The forum feed should be managed by its own angular directive that would then be called in the homepage template, not in the profiles.

Reply:
It is not completely illogical that this function is in this file, especially if we add other information flows in the future I suppose is the feed in the broad sense :).

So i add the test before the this.getLatestTopics_();.